test/integration.test.js

```diff
@@ -182,17 +181,8 @@ describe('registry-data (integration)', function () {
         assume(result.version).eql(BuildFixture.version);
         assume(result.fingerprints.sort()).eql(BuildFixture.fingerprints.sort());
         assume(result.artifacts.sort()).eql(BuildFixture.artifacts.sort());
-        assume(result.recommended).to.be.an('array');
-
-        result.fetchFiles(function (error, files) {
-          assume(error).is.falsey();
-          assume(files).is.an('array');
-          files.forEach(function (file) {
-            assume(file).is.an('object');
-            assume(file.url).is.a('string');
-          });
-          done();
-        });
+        assume(result.recommended.sort()).eql(BuildFixture.recommended.sort());
```

Comment:
does that `.eql` work? these are arrays, so `.eql` by default would be doing a reference equality, right? should this be `.deep.equals`?

Reply:
@msluther `equals` is shallow `eql` is shorthand for deep equals